src/components/Button.js

```diff
@@ -277,4 +280,7 @@ class Button extends Component {
 Button.propTypes = propTypes;
 Button.defaultProps = defaultProps;
 
-export default Button;
+export default compose(
+    withNavigationFallback,
+    withNavigationFocus,
```

Comment:
What does the withNavigationFocus do here?

Reply:
Oh, I forgot to push the final commit. Good catch.

Reply:
I'm still not seeing how it is used, could you explain please?

Reply:
this HOC gives is a new prop called `isFocused` and we are using that to determine if the button is visible on the currently focused/active screen. This way all the buttons on the backgrounded screens will not trigge with `enter` press.